cluster-scope/base/clusterroles/observatorium-operator/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+---
+# TODO: Ideally this can be replaced by a observatorium-operator subscription kind resource
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - https://raw.githubusercontent.com/observatorium/operator/03be03063a5cfebb0bf4fbbc8faca3f252bf5006/manifests/cluster_role.yaml
```

Comment:
@tumido the observatorium operator kustomization is already available in the [apps repo](https://github.com/operate-first/apps/blob/master/observatorium/base/operator/kustomization.yaml). Can you either remove the Observatorium stuff from this PR or remove the existing observatorium kustomization from the apps repo?

Reply:
@4n4nd this is a WIP pr. It's not even following the ADR properly... The ADR is result of me trying things for this PR.

This PR will need to be changed a lot before it's ready to be reviewed.

Reply:
oops my bad, I didn't see the wip title sorry :see_no_evil: